File: 8 KYU/MultiplicationTable.java
public class MultiplicationTable {
	public static String multiTable(int num) {
		// Original problem at https://www.codewars.com/kata/5a2fd38b55519ed98f0000ce
      
		// Declaring variables
		String returned = "";
		int multiplied;
      
		// Adding values to the string
		for (int min = 1; min <= 10; min++) {
			multiplied = min * num;
			returned = returned + "\n" + min + " * " + num + " = " + multiplied;
		}
      
		// Trimming and returning string
		returned = returned.trim();
		return returned;
	}
}

File: 8 kyu/MultiplicationTable.py
# https://www.codewars.com/kata/5a2fd38b55519ed98f0000ce
# 8 KYU

def multi_table(number):
    # Declaring what is going to be returned
    returned = ""
    
    # For loop to add all the lines of the multiplication table
    for min in range(11):
        if min != 0:
            multiplied = min * number
            returned = returned + "\n" + str(min) + " * " + str(number) + " = " + str(multiplied)
    
    # Returning string
    return str(returned.strip())